Pipfile

```diff
@@ -48,6 +48,7 @@ streamz = "==0.6.2"
 wakepy = "==0.5.0"
 werkzeug = "==2.0.3"  # Tanner (4/7/22): pinned for flask
 xem-wrapper = "==0.3.0"
+mantarray-desktop-app = {editable = true, path = "."}
```

Comment:
I don't think we want this here. If you added this for local development, the correct way to do it is to just run `pipenv run pip install -e .` similar to what we do for development in other repos

Reply:
I ran the commands from confluence to update my repo. This was added from the script.